src/app/flower-test/flower-test.component.ts

```diff
@@ -9,141 +10,92 @@ import { FlowerLayoutItem } from '../flower-layout-item/flower-layout-item.compo
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/trifolium repens.png',
-      alt: 'test',
-      active: true,
-      scale: 1.1
+      species: allFlowerSpecies.prunus_americana.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/vaccinium angustifolium.png',
-      alt: 'test',
-      active: true,
-      scale: 1.5
+      species: allFlowerSpecies.rubus_occidentalis.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/rudbeckia hirta.png',
-      alt: 'test',
-      active: true,
-      scale: 2.1
+      species: allFlowerSpecies.rudbeckia_hirta.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/taraxacum officinale.png',
-      alt: 'test',
-      active: true,
-      scale: 1.1
+      species: allFlowerSpecies.solidago_rigida.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/solidago rigida.png',
-      alt: 'test',
-      active: true,
-      scale: 1.3
+      species: allFlowerSpecies.taraxacum_officinale.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/sunflower.png',
-      alt: 'test',
-      active: true,
-      scale: 1.5
+      species: allFlowerSpecies.trifolium_repens.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/black raspberry.png',
-      alt: 'test',
-      active: true,
-      scale: 0.9
+      species: allFlowerSpecies.vaccinium_angustifolium.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/trifolium repens.png',
-      alt: 'test',
-      active: true,
-      scale: 1.1
+      species: allFlowerSpecies.asclepias_syriaca.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/vaccinium angustifolium.png',
-      alt: 'test',
-      active: true,
-      scale: 1.5
+      species: allFlowerSpecies.cirsium_discolor.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/sunflower.png',
-      alt: 'test',
-      active: true,
-      scale: 1.5
+      species: allFlowerSpecies.echinacea_angustifolia.id,
+      blooming: true
     },
     {
-      imgSrc: 'assets/images/1000w-8bit/flowers/black raspberry.png',
-      alt: 'test',
-      active: true,
-      scale: 0.9
-    }
-  ]
+      species: allFlowerSpecies.helianthus_maximiliani.id,
+      blooming: true
+    },
+  ];
+
+  allFlowerSpeciesArray = Object.values(allFlowerSpecies);
 
   constructor() { }
 
   ngOnInit() {
   }
 
+  getFlowers(inputs: {species: string, blooming: boolean}[]): FlowerLayoutItem[] {
+    return inputs.map(({species, blooming}) => {
+      const speciesObj = allFlowerSpecies[species];
+      return {
+        imgSrc: `assets/art/500w/flowers/${speciesObj.art_file}`,
+        alt: speciesObj.name,
+        active: blooming,
+        scale: speciesObj.relative_size
+    }
```

Comment:
```suggestion
      };
```
Just another indentation thing, don't mind me :stuck_out_tongue: